apidef/oas/server.go

```diff
@@ -171,3 +171,20 @@ func (ppk PinnedPublicKeys) ExtractTo(publicKeys map[string]string) {
 		publicKeys[publicKey.Domain] = strings.Join(publicKey.List, ",")
 	}
 }
+
+type Domain struct {
+	// Enabled allow/disallow the usage of the domain
```

Comment:
GoDocs should end with `.` Also, can you please use the same pattern with other enabled fields?